### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by BradyMitch**
## 🎯 Summary

- Replaced green checkmark emoji to be more esthetically pleasing.
- Updated hidden comments to make it easier to read and edit by separating the Jira Ticket and description lines. 

## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/Code_of_Conduct.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - My changes generate no new warnings.

### CODE REVIEW

**Comment by NoorChasib** [comment]
the check box is grey now, arguably less aesthetically pleasing...

**Comment by BradyMitch** [comment]
> the check box is grey now, arguably less aesthetically pleasing...

@NoorChasib Which checkbox is grey? I don't see any grey in the checkbox.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
-<!-- Provide an explanation of your changes and -->
-<!-- include the JIRA ticket link               -->
+<!-- ADD JIRA LINK BELOW -->  
 
-## ✅ Checklist
+<!-- PROVIDE BELOW an explanation of your changes and any images to support your explanation -->
+
+
+## ☑️ Checklist
```
Comment (BradyMitch) [inline .github/pull_request_template.md]:
```suggestion
## 🔰 Checklist
```

**Review by Bottle7 — APPROVED** [review]

**Comment by BradyMitch** [comment]
> > the check box is grey now, arguably less aesthetically pleasing...
> 
> @NoorChasib Which checkbox is grey? I don't see any grey in the checkbox.

Updated.

**Review by dbarkowsky — APPROVED** [review]

**Review by NoorChasib — APPROVED** [review]

**Pull request merged**